tools/pioasm/python_output.cpp

```diff
@@ -61,7 +61,9 @@ struct python_output : public output_format {
         FILE *out = open_single_output(destination);
         if (!out) return 1;
 
-        header(out, "This file is autogenerated by pioasm; do not edit!");
+        std::stringstream header_string;
+        header_string << "This file is autogenerated by pioasm version " << PIOASM_VERSION_STRING << "; do not edit!";
+        header(out, header_string.str());
```

Comment:
This code-snippet is identical in each of the different output-format generators - perhaps it's worth abstracting out a new common function, and then this could be simplified to:
```c
        std::stringstream header_string = standard_header_comment();
        header(out, header_string.str());
```
(and I guess it would also mean that each of the different output-format generators don't need to separately include `version.h`)

Reply:
(Tangential unimportant observation: The ADA generator's `header()` function has an `indent` parameter, which none of the `header()` functions for the other language generators have! :slightly_smiling_face: )

Reply:
I saw both of these, and decided i didn't care that much ;-)